ui/app/router.js

```diff
@@ -13,6 +13,7 @@ Router.map(function() {
       this.route('auth');
       this.route('init');
       this.route('logout');
+      this.mount('open-api-explorer', { path: '/api-explorer' });
```

Comment:
is there anything preventing the user from simply navigating to `/api-explorer` in their browser? i think it makes sense to only navigate to the page by using the web cli, but i was just curious. 😄

Reply:
Nope! That's what we want though since you can pass around the URLs (with ?filter) to share specific views. If a route or engine is in this map, the URLs can be navigated to - the only place where we've set it up so that you could only load an engine if some logic is true is in the application initializer https://github.com/hashicorp/vault/blob/8f6d2002f89c73a59c1c43782274e0b2f03f80e9/ui/app/initializers/enable-engines.js but even there we just mount the engine (for now - we've talked about having some logic that would skip that). We could also do logic here in the router.js file, but there's no access to hooks that would pause the application's startup like there are in initializers.